src/DataMapper/ScheduleMapper.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GroupLife\Core\DataMapper;
+
+use GroupLife\Core\Schedule;
+
+class ScheduleMapper
```

Comment:
Hard to review: could you make this PR to contain only new changes related to VisitorMapper?

Reply:
My fault. Already corrected